plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1727,6 +1728,18 @@ SELECT COUNT(*)
                 WHERE double_col NOT IN (0.0, -16.33, -17.33)"""))
                 .matches("VALUES (BIGINT '7')")
                 .isNotFullyPushedDown(FilterNode.class);
+
+        assertThat(query("SELECT price, vendor FROM " + JSON_TABLE + " WHERE price IS NOT NULL"))
```

Comment:
This table has null handling disabled, should it check for all values? and/or just use the count query below?